.circleci/config.yml

```diff
@@ -330,12 +330,6 @@ jobs:
       - run:
           name: Install Node Modules
           command: |
-                # The release of nodejs currently being used by Calypso includes a buggy version
```

Comment:
Was this resolved?

Reply:
We're no longer compiling `electron-spellchecker`, so this workaround for node-gyp isn't necessary. We can always revive it if necessary. But for now it's additional complexity we don't need.